ui/src/app/job-list/table/table.component.spec.ts

```diff
@@ -168,6 +168,25 @@ describe('JobsTableComponent', () => {
       .toEqual("Precondition Failed (412): Job already in terminal status `FAILED` Dismiss");
   }))
 
+  it('should only show length for exhaustive job streams', async(() => {
```

Comment:
nit: this could be two unit tests

Reply:
I wanted to explicitly cover the transition between non-exhaustive -> exhaustive to make sure the text updated appropriately. Added a comment.